zerver/openapi/zulip.yaml

```diff
@@ -2058,12 +2059,93 @@ paths:
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/JsonSuccess'
+                allOf:
+                - $ref: '#/components/schemas/JsonSuccess'
+                - example:
+                      {
+                          "msg": "",
+                          "result": "success"
+                      }
+        '400':
+          description: Bad request.
+          content:
+            application/json:
+              schema:
+                allOf:
+                - $ref: '#/components/schemas/JsonError'
+                - example:
+                    {
+                        "code": "BAD_REQUEST",
+                        "msg": "Invalid stream id",
+                        "result": "error"
+                    }
+    patch:
+      description: Update the stream with the given ID.
+      parameters:
+      - name: stream_id
+        in: path
+        description: The ID of the stream to be updated.
+        schema:
+          type: integer
+        example: 42
+        required: true
+      - name: description
+        in: query
+        description: The new description for the stream.
+        schema:
+          type: string
+        example: "This stream is related to football dicsussions."
+        required: false
+      - name: new_name
+        in: query
+        description: The new name for the stream.
+        schema:
+          type: string
+        example: "Italy"
+        required: false
+      - name: is_private
+        in: query
+        description: The new state of the stream.
+        schema:
+          type: boolean
+        example: true, false
```

Comment:
These should just have a single example value, e.g. `true`; this rendered as the string `'true, false'`, which is not a valid boolean.